### ISSUE TITLE
Ruby warnings on 1.9.3

### ISSUE THREAD

**Issue by cgriego** [wvanbergen/request-log-analyzer#124]
I'm using @scoutapp and some of its plugins use the request-log-analyzer gem. Scout runs with Ruby warnings enabled. Unfortunately request-log-analyzer seems to generate a lot of Ruby warnings under MRI 1.9.3.

```
/usr/local/rvm/gems/ruby-1.9.3-p194@scout/gems/request-log-analyzer-1.12.4/lib/request_log_analyzer/aggregator/database_inserter.rb:38: warning: assigned but unused variable - v
/usr/local/rvm/gems/ruby-1.9.3-p194@scout/gems/request-log-analyzer-1.12.4/lib/request_log_analyzer/file_format.rb:290: warning: assigned but unused variable - name
/usr/local/rvm/gems/ruby-1.9.3-p194@scout/gems/request-log-analyzer-1.12.4/lib/request_log_analyzer/file_format.rb:290: warning: assigned but unused variable - name
/usr/local/rvm/gems/ruby-1.9.3-p194@scout/gems/request-log-analyzer-1.12.4/lib/request_log_analyzer/file_format.rb:300: warning: shadowing outer local variable - name
/usr/local/rvm/gems/ruby-1.9.3-p194@scout/gems/request-log-analyzer-1.12.4/lib/request_log_analyzer/log_processor.rb:55: warning: shadowing outer local variable - file
/usr/local/rvm/gems/ruby-1.9.3-p194@scout/gems/request-log-analyzer-1.12.4/lib/request_log_analyzer/output/fixed_width.rb:167: warning: assigned but unused variable - fill_column
/usr/local/rvm/gems/ruby-1.9.3-p194@scout/gems/request-log-analyzer-1.12.4/lib/request_log_analyzer/tracker/numeric_value.rb:71: warning: assigned but unused variable - info
/usr/local/rvm/gems/ruby-1.9.3-p194@scout/gems/request-log-analyzer-1.12.4/lib/request_log_analyzer/tracker/numeric_value.rb:295: warning: assigned but unused variable - name
/usr/local/rvm/gems/ruby-1.9.3-p194@scout/gems/request-log-analyzer-1.12.4/lib/request_log_analyzer/tracker/numeric_value.rb:300: warning: assigned but unused variable - name
/usr/local/rvm/gems/ruby-1.9.3-p194@scout/gems/request-log-analyzer-1.12.4/lib/request_log_analyzer/aggregator/echo.rb:15: warning: assigned but unused variable - v
/usr/local/rvm/gems/ruby-1.9.3-p194@scout/gems/request-log-analyzer-1.12.4/lib/request_log_analyzer/aggregator/summarizer.rb:137: warning: assigned but unused variable - key
```

**Comment by wvanbergen** [comment]
I eliminated all the warnings that are generated by a basic run of the application.

**Comment by wvanbergen** [comment]
Included in the latest version: 1.12.5

**Comment by cgriego** [comment]
Thanks! That fixed the warnings I was seeing. :+1:
